packages/block-library/src/query-pagination/index.php

```diff
@@ -5,12 +5,36 @@
  * @package WordPress
  */
 
+/**
+ * Renders the `core/query-pagination` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ *
+ * @return string Returns the wrapper for the Query pagination.
+ */
+function render_block_core_query_pagination( $attributes, $content ) {
+	$wrapper_attributes = get_block_wrapper_attributes();
+
+	if ( empty( $content ) || ctype_space( $content ) ) {
```

Comment:
For some reason the `$content` is a long string of whitespace when it's empty, so I added the `ctype_space` condition.